### ISSUE TITLE
Bug: CORS is blocking OPTIONS requests

### ISSUE THREAD

**Issue by IzMichael** [dicedtomatoreal/typex#233]
### What happened?

Hiya, I've just installed Zipline and I'm trying to use the API to upload some images from a web page, but I'm running into a CORS issue.

The preflight request is returning a 405 Method Not Found
![image](https://user-images.githubusercontent.com/57115702/203691924-2ed19f74-888f-48a5-92e3-1767839d4299.png)

### Version

latest (ghcr.io/diced/zipline:latest)

### What browser(s) are you seeing the problem on?

Chromium-based (Chrome, Edge, Brave, Opera, mobile chrome/chromium based, etc)

### Browser Logs

```shell
Access to fetch at 'https://REDACTED/api/upload' from origin 'http://localhost:57640' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: It does not have HTTP ok status.

POST https://REDACTED/api/upload net::ERR_FAILED
```

### Additional Info

X-Post from Discord forum (https://discord.com/channels/729771078196527176/1045123801312149555)

**Comment by diced** [comment]
seems to be blocking OPTIONS, oversight when rewriting the API stuff, fixed in 1b6af9f

**Comment by IzMichael** [comment]
ty!
